sdk/typescript/src/types/validator.ts

```diff
@@ -176,13 +178,23 @@ export const SuiSystemStateSummary = object({
   safe_mode: boolean(),
   epoch_start_timestamp_ms: number(),
   min_validator_stake: number(),
-  max_validator_candidate_count: number(),
+  max_validator_count: number(),
   governance_start_epoch: number(),
   stake_subsidy_epoch_counter: number(),
   stake_subsidy_balance: number(),
   stake_subsidy_current_epoch_amount: number(),
   total_stake: number(),
   active_validators: array(SuiValidatorSummary),
+  pending_active_validators_id: string(),
+  pending_active_validators_size: number(),
+  pending_removals: array(number()),
```

Comment:
What do the numbers in this array represent?

Reply:
It's the index of the validator in the active validator array that will be removed

Reply:
Got it. A little weird that it's indexes and not object IDs or something like that but that's fine.

Reply:
Agreed. It's some very old code. If we end up having time, we may try to update it latter

Reply:
Added doc comments